tests/objc-cli/libmanaged/Tests/Tests.m

```diff
@@ -90,10 +90,6 @@ - (void)testConstructors {
 
 	id super_unique_default_init = [[Constructors_SuperUnique alloc] init];
 	XCTAssert ([super_unique_default_init id] == 411, "super id");
-
-	// FIXME - this should not be allowed, that .ctor is not available to call in .NET as it is not re-declared in SuperUnique
```

Comment:
We will need to do some error/warning tests (like touch tests) to ensure we don't regress things we can't execute. OTOH that can be done in a separate PR.